src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = body.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+promise1.then(() => {
+  const successMessage = document.createElement('div');
+
+  successMessage.classList.add('message');
+  successMessage.textContent = 'Promise was resolved!';
+  body.append(successMessage);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('reject'));
```

Comment:
```suggestion
    reject('Promise was rejected!');
```

Reply:
Eslint gives me an error:
Expected the Promise rejection reason to be an Error.